azure/template.json

```diff
@@ -79,12 +79,431 @@
+            "type": "string"
+        },
+        "certificateName": {
+            "type": "string"
+        },
+        "internalApiSandboxAppServiceName": {
+            "type": "string"
+        },
+        "internalApiSandboxCustomHostName": {
+            "type": "string"
+        },
+        "sandBoxFunctionAppNames": {
+            "type": "array"
+        },
+        "sharedSqlResourceGroup": {
+            "type": "string"
+        },
+        "secondaryDatabaseName": {
+            "type": "string"
+        },
+        "sharedSqlServerName": {
+            "type": "string"
+        },
+        "databaseSkuName": {
+            "type": "string"
+        },
+        "databaseTier": {
+            "type": "string"
+        },
+        "logAnalyticsSubscriptionId": {
+            "type": "string"
+        },
+        "logAnalyticsResourceGroup": {
+            "type": "string"
+        },
+        "logAnalyticsWorkspaceName": {
+            "type": "string"
+        },
+        "externalActiveApiSandboxCustomHostname": {
+            "type": "string"
         }
     },
     "variables": {
         "InstanceName": "[if(equals(toUpper(parameters('environmentName')),'PROD'),'',parameters('environmentName'))]"
     },
     "resources": [
+        {
+            "apiVersion": "2017-05-10",
+            "name": "sandbox-app-service-plan",
+            "type": "Microsoft.Resources/deployments",
+            "condition": "[equals(parameters('deploySandbox'), 'true' )]",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/app-service-plan.json",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "appServicePlanName": {
+                        "value": "[parameters('activeSandboxAppServicePlanName')]"
+                    },
+                    "aseHostingEnvironmentName": {
+                        "value": "[parameters('aseHostingEnvironmentName')]"
+                    },
+                      "aseResourceGroup": {
+                        "value": "[parameters('aseResourceGroup')]"
+                    },
+                    "aspLocation": {
+                        "value": "[resourceGroup().location]"
+                    },
+                    "aspSize": {
+                        "value": "[parameters('aspSize')]"
+                    },
+                    "aspInstances": {
+                        "value": "[parameters('aspInstances')]"
+                    }
+                }
+            }
+        },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "[concat('AppService-', parameters('externalApiActiveSandboxAppServiceName'))]",
+            "type": "Microsoft.Resources/deployments",
+            "condition": "[equals(parameters('deploySandbox'), 'true')]",
```

Comment:
"condition" takes a bool value, so this can just be set to [parameters('deploySandbox')] now